target_chains/ethereum/sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/pyth-evm-js",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Why do you need to bump the version

Reply:
This is because the index file will change. @cctdaniel I think you missed to add the contract address here.

Reply:
great catch ser